git-branchless-lib/src/core/rewrite/rewrite_hooks.rs

```diff
@@ -148,19 +133,6 @@ pub fn hook_post_rewrite(
     let event_log_db = EventLogDb::new(&conn)?;
     let event_tx_id = event_log_db.make_transaction_id(now, "hook-post-rewrite")?;
 
-    {
```

Comment:
Is there code that also writes the deferred commits that is now also obsolete? If so, we should delete it too. Or, alternatively, keep all the code but comment out the `add_events` line with a note, as we might need similar code in the future to fix certain other bugs.